server/server.js

```diff
@@ -161,6 +162,10 @@ export default class TONQServer {
         });
         this.server.setTimeout(2147483647);
 
+        const version = packageJson().version;
+        const startCounter = new StatsCounter(this.stats, STATS.start, [`version=${version}`]);
```

Comment:
Format of tag is incorrect. Ref:
https://github.com/sivy/node-statsd/blob/master/lib/statsd.js#L215
https://github.com/prometheus/statsd_exporter#tagging-extensions
node-statsd uses DogStatsD format, that is `|#tag:val,tag2:val2`
The code writes in format `|#` `tag=val` that is not correct and recognized.